instructions/docs/add-the-encryption-sdk.md

```diff
@@ -143,16 +142,9 @@ Now that you are encrypting data before storing it, you need to decrypt it befor
 // Save and Exit
 ```
 
-#### What Just Happened
 ```Python
 # Edit src/document_bucket/api.py
-
-    def retrieve(
-            self,
-            pointer_key: str,
-            expected_context_keys: Set[str] = set(),
-            expected_context: Dict[str, str] = {},
-    ) -> DocumentBundle:
+# Find the retrieve function and edit it...
```

Comment:
consider "Find the `retrieve` function and edit it to add the following parameters:"

Reply:
Yep! Added with slight edit